rir/src/utils/filesystem.cpp

```diff
@@ -0,0 +1,28 @@
+#include "filesystem.h"
+#include <errno.h>
+#include <ftw.h>
+#include <stdio.h>
+#include <unistd.h>
+
+// From
+// https://stackoverflow.com/questions/5467725/how-to-delete-a-directory-and-its-contents-in-posix-c
+static int unlink_cb(const char* fpath, const struct stat* sb, int typeflag,
```

Comment:
naming convention is camelCase

Reply:
yes that code was copy/pasted from StackOverflow. I admit I was lazy and didn't want to write the "delete directory" function manually :)